contrib/nitrpg/src/statistics.nit

```diff
@@ -78,6 +69,97 @@ redef class Player
 	end
 end
 
+redef class Counter[E]
+	redef fun inc(e) do
+		self[e] = self[e] + 1
+	end
+
+	# Decrements the value of `key` statistic entry by 1.
+	fun dec(key: String) do
```

Comment:
Yeah... I'm not sure of that since the semantics of `dec` on unexisting key is quite specific to Nitrpg.